src/main/resources/application.properties

```diff
@@ -4,8 +4,3 @@ spring.jpa.hibernate.ddl-auto=update
 spring.jpa.properties.hibernate.format_sql=true
 logging.level.org.hibernate.SQL=DEBUG
 logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
-security.authentication.jwt.expirationtime=86400000
-security.authentication.jwt.secret=switchboardsecret
-security.authentication.superuser.username=admin
-security.authentication.superuser.password=admin
```

Comment:
I think you might be able to do `${JWT_SECRET}` to get the environment variables. Then you could copy this file and name it `application-development.properties` so that we don't have to define those env variables for local development.

Reply:
You don't need to set env variables for local development. Defaults will be used, if no env variables are set.